gnat2goto/driver/tree_walk.adb

```diff
@@ -694,31 +694,61 @@ package body Tree_Walk is
       Resolved_Underlying : constant Irep :=
         Follow_Symbol_Type (Underlying, Global_Symbol_Table);
       --  ??? why not get this from the entity
+
+      function Get_Bound_Symbol (Bound_Node : Node_Id) return Bound_Type_Symbol
```

Comment:
This function (seemingly?) only works with arrays, but the name doesn't make this clear at the moment.

Reply:
Changed to `Get_Array_Attr_Bound_Symbol`.